view/view_test.go

```diff
@@ -369,7 +369,7 @@ LOREM IPSUM LOREM IPSUM LOREM IPSUM`
 
 	res := trimOutput(output, 84, 7)
 
-	expectedRes := `LOREM  IPSUM ♥ LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM
+	expectedRes := `LOREM  IPSUM ♥ LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM I
```

Comment:
I'm guessing this change is because it won't handle some emoji-related edge cases with 100% precision now?